### ISSUE TITLE
Kiwix freezes whenever I try to download Wiki.

### ISSUE THREAD

**Issue by CrimsonicSniper** [kiwix/kiwix-desktop#1228]
![Image](https://github.com/user-attachments/assets/412eaf77-162e-4498-9959-3a013af8f5e5)
This is the stats of my PC, I have a 1tb hdd attached to it which im trying to download Wikipedia, Wikihow and some others to. Whenever I try to download Wikipedia, kiwix freezes. This issue is very annoying and I do not know how to fix it. I have tried to kill the task and open another instance in a different directory and now when I checked the roaming/kiwix folder it had about 50 gigs of useage. I had set the directory to G:/kiwix and I do not know why it was downloading there in the first place. 
![Image](https://github.com/user-attachments/assets/46638433-a74e-446b-a1f5-9bdd1490a142)
![Image](https://github.com/user-attachments/assets/d90ca12d-04d8-4fb5-a507-bdfcd57fd639)
I am not that good at software so please dumb it down for me. Any help would be appreciated

**Comment by CrimsonicSniper** [comment]
I also cannot delete the large 26GB files because somehow the explorer doesnt want to and they have reappeared in my system.

**Comment by CrimsonicSniper** [comment]
![Image](https://github.com/user-attachments/assets/d599cb44-f40a-44da-aa5a-8149a5959984)

**Comment by kelson42** [comment]
@veloman-yunkan Might that be a duplicate of #87?

**Comment by veloman-yunkan** [comment]
> [@veloman-yunkan](https://github.com/veloman-yunkan) Might that be a duplicate of [#87](https://github.com/kiwix/kiwix-desktop/issues/87)?

@kelson42 I bet it is

**Comment by CrimsonicSniper** [comment]
no, the #87 the download doesnt start and the other things work fine but in my problem the entire window freezes, there is also not a fix in 87

**Comment by CrimsonicSniper** [comment]
also its been 4 years how is this not fixed

**Comment by kelson42** [comment]
@veloman-yunkan I will close the issue but if https://github.com/kiwix/kiwix-build/issues/757 would be fixed, then @CrimsonicSniper would be able to check and enjoy the fix himself with the nightly instead of waiting the 2.4.0 release.
